superset-frontend/src/dashboard/components/Header/HeaderActionsDropdown/index.jsx

```diff
@@ -136,6 +136,12 @@ class HeaderActionsDropdown extends React.PureComponent {
       });
   }
 
+  UNSAFE_componentWillReceiveProps(nextProps) {
+    if (this.props.customCss !== nextProps.customCss) {
+      this.setState({ css: nextProps.customCss });
```

Comment:
seems like `customCss` comes from redux store, which is updated via `updateCss`.
Maybe we can try removing the following from `changeCss`, and inject the custom css after saving it here?:
```js
this.setState({ css }, () => {
  injectCustomCss(css);
});
```

That would leave us with a single source of truth, which is better (if it works as desired)